### DISCUSSION TITLE
Hi, CEBRA team! Is that make sense to use original data not fitted by cebra model to calculate the Consistency across features?

### DISCUSSION THREAD

**Discussion by qweasdf1354** [AdaptiveMotorControlLab/CEBRA#115] [Q&A]
Hi, author! I using the cebra for month and today i found use original data not fitted by cebra model to calculate the Consistency across features the results seems good for me and the fitted embedding seems have low consistency scores.

I use single train model by 2 continuous and 1 discrete variables, the maxiteration i set is 10000 which is not sufficient to get the good consistency results and embedding.
So, I set the maxiteration  = 50000 now and batch_size=10240 which take me 25min to train a single model. It is still training now(total 54 model)

I want to know  is that make sense to use original data not fitted by cebra model to calculate the Consistency across features?

Thanks a lot!

**Comment by MMathisLab** [comment]
hi @qweasdf1354 I am not sure exactly the issue and goal; you are asking how can you use consistency without training with cebra? 

The idea of consistency is to (1) be sure you get consistent results, which would imply a good model. So you should train the model the same way several times to measure this (see figure 1); (2) to use as a metric of differences across datasets (see Figure 6).

**Comment by stes** [comment]
Hi @qweasdf1354 , any further question/follow up needed here?
